crates/utils/src/request.rs

```diff
@@ -138,20 +138,20 @@ pub(crate) async fn fetch_pictrs(
       .map_err(|e| RecvError(e.to_string()))?;
 
     if response.msg == "ok" {
-      Ok(Some(response))
+      Ok(response)
     } else {
       Err(anyhow!("{}", &response.msg).into())
     }
   } else {
-    Ok(None)
+    Err(anyhow!("pictrs_url not set up in config").into())
```

Comment:
I dont think this should be an error, some people might want to host Lemmy without images (even if that isnt really supported by lemmy-ui).

Reply:
That's just for the `fetch_pictrs` function. The `fetch_site_metadata_and_pictrs_data` is the public function, which does coerce any errors to `None`.

Reply:
Ah missed that, I agree its better.